ci-operator/config/openshift/release/openshift-release-master__ci-4.11.yaml

```diff
@@ -230,6 +230,11 @@ tests:
       BASE_DOMAIN: ci2.azure.devcluster.openshift.com
       TEST_TYPE: upgrade-conformance
     workflow: openshift-upgrade-azure-single-node
+- as: e2e-metal-ipi-ovn-ipv4
+  interval: 48h
+  steps:
+    cluster_profile: equinix-ocp-metal
+    workflow: baremetalds-e2e-ovn-ipv4
```

Comment:
An additional job per 48h hopefully won't hit our quota too hard. Do you want to do one for 4.12 as well?

Reply:
yeah, good idea. PR updated with that. I think 4.12 wasn't even around when I originally made this change back in May.